show/main.py

```diff
@@ -520,6 +741,52 @@ def uptime():
     """Show system uptime"""
     run_command('uptime -p')
 
+@cli.command()
+def clock():
+    """Show date and time"""
+    run_command('date')
+
+@cli.command('system-memory')
+def system_memory():
+    """Show memory information"""
+    comm="free -m"
```

Comment:
Change to `command = "..."` for consistency ("comm" -> "command" and add spaces around equals sign).